taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -0,0 +1,73 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from "@mui/material";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import FetchRelations from "./FetchRelations";
+import ListAllEntryProperties from "./ListAllEntryProperties";
+import ListAllOtherProperties from "./ListAllOtherProperties";
+
+const AccumulateAllComponents = ({ incomingNodeObject, id, isEntry, url }) => {
+    const [nodeObject, setNodeObject] = useState(null);
+    const [open, setOpen] = useState(false);
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        setNodeObject(incomingNodeObject);
+    }, [incomingNodeObject])
+
+    const handleClose = () => {setOpen(false)};
+    const handleBack = () => {navigate('/entry')};
+
+    const handleSubmit = () => {
+        delete nodeObject['id']; // ID not allowed in POST
```

Comment:
not the best thing to modify a state directly, it may lead to some unexpected behavior. What about using something like `const {id, ...data} = nodeObject` and you pass the `data`